osu.Server.Spectator/Hubs/Multiplayer/Matchmaking/MatchmakingMatchController.cs

```diff
@@ -135,7 +135,7 @@ public async Task HandleUserJoined(MultiplayerRoomUser user)
             switch (state.Stage)
             {
                 case MatchmakingStage.WaitingForClientsJoin:
-                    if (++joinedUserCount == room_size)
+                    if (++joinedUserCount >= room_size)
```

Comment:
I think this should be safe...

The original change I was going to go with was storing the room settings to an `IMemoryCache` object, and then pulling those settings in this class. This is because otherwise I have no way to pass the MM room size between classes due to the database round-trip.

Reply:
That is true, though this process should happen fairly quick (~1 second) so I don't know if it's a problem at least for testing.

This code only starts triggering after every player has accepted the invitation, so they've already signaled their intent to join and the only thing remaining is the client to invoke `JoinRoom()`.

Reply:
So they can still join during warmup?

Reply:
With room size zero, yes.